packages/components/simple-table/src/simple-table.spec.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render } from '../../../../src/test-utils';
+import SimpleTable from '.';
+
+const testRows = [
+  { id: '1-parasite', title: 'Parasite', year: 2019 },
+  { id: '2-woman', title: 'Woman At War', year: 2018 },
+  { id: '3-gems', title: 'Uncut Gems', year: 2019 },
+];
+
+const testColumns = [
+  {
+    key: 'title',
+    label: 'Title',
+  },
+  {
+    key: 'year',
+    label: 'Year',
+  },
+];
+
+const baseProps = { rows: testRows, columns: testColumns };
+
+describe('SimpleTable', () => {
+  it('should forward data-attributes', () => {
+    const { container } = render(<SimpleTable {...baseProps} data-foo="bar" />);
+    expect(container.querySelector("[data-foo='bar']")).toBeInTheDocument();
+  });
+
+  it('should render the column labels', () => {
+    const { queryByText } = render(<SimpleTable {...baseProps} />);
```

Comment:
Below we save the return in `rendered` here not. Should we also use `const rendered = ` here?